docs/monitoring.md

```diff
@@ -1,20 +1,30 @@
 ---
-title: Monitoring Kafka on Kubernetes
+title: Monitoring Apache Kafka on Kubernetes
 shorttitle: Monitoring
 weight: 600
 ---
 
 
 
-This documentation shows you how to enable custom monitoring on a Kafka cluster installed using the [Kafka operator](/products/kafka-operator/).
+This documentation shows you how to enable custom monitoring on an Apache Kafka cluster installed using [{{< kafka-operator >}}](https://github.com/banzaicloud/koperator).
 
 ## Using Helm for Prometheus
 
-By default operator installs Kafka Pods with the following annotations, also it opens port 9020 in all brokers to enable scraping.
+By default, the {{< kafka-operator >}} does not set annotations on the broker pods. To set annotations on the broker pods, specify them in the KafkaCluster CR. Also, you must open port 9020 on brokers and in CruiseControl to enable scraping. For example:
 
 ```yaml
-    "prometheus.io/scrape": "true"
-    "prometheus.io/port":   "9020"
+brokerConfigGroups:
+  default:
+    brokerAnnotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "9020"
+
+...
```

Comment:
**Recommended**: IMO we **SHOULD** use a comment for ellipsis in order to be YAML compliant and still signal redacted content.
```suggestion
# ...
```